src/pages/homePage.tsx

```diff
@@ -1,10 +1,7 @@
-import { Outlet } from "react-router-dom";
-
 export default function HomePage() {
   return (
     <div className="flex justify-center items-center flex-wrap h-screen">
       <p>HomePage</p>
-      <Outlet />
```

Comment:
Waarom is het Outlet-component verwijderd?

Reply:
Omdat die geen zin had. ik was dit vergeten we te halen.